components/src/core/components/List/list.scss

```diff
@@ -112,6 +117,11 @@
         }
       }
     }
+    :deep(.oxd-table-cell:has(.oxd-checkbox-input)) {
+      &:first-child {
```

Comment:
```css
:deep(.oxd-table-th:has(.oxd-checkbox-input), .oxd-table-cell:has(.oxd-checkbox-input)) {
  &:first-child {
    padding-right: 0;
  }
}
```

reduces the repetition

Reply:
when i merge like this, only the first selector gets affected.

Reply:
:deep(.oxd-table-th:has(.oxd-checkbox-input)), :deep(.oxd-table-cell:has(.oxd-checkbox-input)) {
      &:first-child {
        padding-right: 0;
      }
    }
    
I applied it this way.